From 88498a86000db0c8c863ea4a31bef97ba6fb4a69 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Thu, 9 Oct 2014 21:00:12 -0400
Subject: [fs] xfs: negate mount workqueue init error value

Message-id: <1412888441-4181-19-git-send-email-sandeen@redhat.com>
Patchwork-id: 97344
O-Subject: [RHEL7.1 PATCH 18/47] [fs] xfs: negate mount workqueue init error value
Bugzilla: 1145837
RH-Acked-by: Brian Foster <bfoster@redhat.com>
RH-Acked-by: Dave Chinner <dchinner@redhat.com>

Bugzilla: 1145837
Upstream Status: Committed upstream
Build Info: https://brewweb.devel.redhat.com/taskinfo?taskID=8082426
Tested: locally using xfstests

Thanks,
-Eric

    commit 45687642e43d5a3b700d6c1df0e07b3976c90c08
    Author: Dave Chinner <dchinner@redhat.com>
    Date:   Thu May 15 09:22:53 2014 +1000

    xfs: negate mount workqueue init error value

    Signed-off-by: Dave Chinner <dchinner@redhat.com>
    Reviewed-by: Jie Liu <jeff.liu@oracle.com>
    Signed-off-by: Dave Chinner <david@fromorbit.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/xfs/xfs_super.c b/fs/xfs/xfs_super.c
index a5b74ff..0bb0095 100644
--- a/fs/xfs/xfs_super.c
+++ b/fs/xfs/xfs_super.c
@@ -1432,7 +1432,7 @@ xfs_fs_fill_super(
  if (error)
   goto out_free_fsname;
 
- error = xfs_init_mount_workqueues(mp);
+ error = -xfs_init_mount_workqueues(mp);
  if (error)
   goto out_close_devices;
 
-- 
1.7.1